types/bitcoin.types.ts

```
@@ -46,21 +47,79 @@
+      path: ["values"],
+    }
+  );
+
+export const depositOptionsSchema = z.object({
+  amount: z.string().refine((val) => !isNaN(Number(val)) && Number(val) > 0, {
+    message: "Amount must be a valid positive number",
+  }),
+  api: z.string().url(),
+  data: z.string().optional(),
+  name: z.string().optional().default(DEFAULT_ACCOUNT_NAME),
+  gateway: z.string(),
```

ESLint

[error] Expected object keys to be in ascending order. 'gateway' should be before 'name'.

(sort-keys-fix/sort-keys-fix)